tests/rkt_fetch_test.go

```diff
@@ -136,7 +136,10 @@ func TestFetchFullHash(t *testing.T) {
 	}
 
 	for _, tt := range tests {
-		hash := importImageAndFetchHash(t, ctx, tt.fetchArgs, imagePath)
+		hash, err := importImageAndFetchHash(t, ctx, tt.fetchArgs, imagePath)
+		if err != nil {
+			t.Skip("remote fetching failed, probably a network failure. Skipping...")
```

Comment:
The image being fetched is `imagePath := getInspectImagePath()`. That's a local image, so not a network failure.